lms/static/scripts/frontend_apps/components/ErrorDialogApp.js

```diff
@@ -12,7 +12,7 @@ import Dialog from './Dialog';
  */
 export default function ErrorDialogApp() {
   const {
-    errorDialog: { errorCode, errorDetails = '' },
+    errorDialog: { errorCode, errorDetails = {} },
```

Comment:
Looking at the `ErrorDisplay` implementation, it looks like the previous default was correct (empty string) but the type definition of `ErrorLike` was incomplete.